com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/v2/CloudModelBuilderTest.java

```diff
@@ -23,7 +23,7 @@
 import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudApplicationExtended;
 import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudServiceExtended;
 import com.sap.cloud.lm.sl.cf.core.cf.HandlerFactory;
-import com.sap.cloud.lm.sl.cf.core.cf.util.DeployedLiveModuleCalculator;
+import com.sap.cloud.lm.sl.cf.core.cf.util.DeployedModuleCalculatorColor;
```

Comment:
The change to `DeployedModuleCalculator` is good but with the `Color` suffix, not so much.
My suggestion is name the concrete class something like `DeployedModuleCalculatorWithColor`